app/SwaggerValidator/HttpRequest.php

```diff
@@ -39,14 +39,19 @@ public function validateAgainst(SwaggerSchema $schema): SwaggerValidationResult
     private static function validateRequestParameterSchema(Request $request, $validator, stdClass $requestParameterSchema)
     {
         $data = (new RequestParameter($request))->getData($requestParameterSchema);
+        $data = json_decode(json_encode($data));
 
         if (isset($requestParameterSchema->schema)) {
             $jsonSchema = $requestParameterSchema->schema;
         } else {
             $jsonSchema = $requestParameterSchema;
         }
 
-        $validator->validate($data, $jsonSchema);
+        $required = isset($requestParameterSchema->required) ? $requestParameterSchema->required : true;
+
+        if ($required || $data !== null) {
```

Comment:
Bo z tego ifa wychodzi, że jak pole nie jest wymagane, ale są jakieś dane, to weryfikuje i tak.